packages/block-library/src/comments-title/index.php

```diff
@@ -33,28 +33,45 @@ function render_block_core_comments_title( $attributes ) {
 		return;
 	}
 
-	$single_default_comment_label = $show_post_title ? __( 'Response to' ) : __( 'Response' );
-	if ( $show_comments_count ) {
-		$single_default_comment_label = $show_post_title ? __( 'One response to' ) : __( 'One response' );
+	if ( $show_comments_count && ! empty( $comments_count ) ) {
```

Comment:
The effect of the `empty()` is that if the comment count is enabled, zero comments will result in `Responses` or `Responses on Post Title`. Due to the `try... catch` in the JavaScript I suspect this could display differently in the editor.

Is that the intent?

Reply:
Maybe the `empty()` was added because it's kind of a counterpart to the [`commentsCount !== undefined` criterion on R110](https://github.com/WordPress/gutenberg/pull/40817/files#diff-067b10489c9ea00ab3948e0ceb4be966ed90f47985a2704239a8859a686978caR110)? cc/ @c4rl0sbr4v0 

FWIW, I [added](https://github.com/WordPress/gutenberg/pull/40817/commits/d1580ec184882e08aca52a9cf66b85d67b0b6fc6) that there in order to avoid an `undefined responses to "Post Title"` flash while loading. (This is only relevant to the Post Editor, where we're fetching the actual comments on the post, whereas in the Site Editor, we're using 3 hard-wired placeholders.)

Anyway, on the frontend, we [bail early](https://github.com/WordPress/gutenberg/pull/40817/files#diff-f1c094255127dfb8013473531eabaa07f7cd0029bc4bdcb90db3b7745d9c2e25R32) if there are no comments.

So I think we can safely remove the `empty()` criterion here

```suggestion
	if ( $show_comments_count ) {
```

(We might need to make sure we're using the correct variable type on [R110](https://github.com/WordPress/gutenberg/pull/40817/files#diff-f1c094255127dfb8013473531eabaa07f7cd0029bc4bdcb90db3b7745d9c2e25R32) if we end up changing `$comments_count`'s definition, [see](https://developer.wordpress.org/reference/functions/get_comments_number/#return).)